### DISCUSSION TITLE
ETA of pool protocol

### DISCUSSION THREAD

**Discussion by sotr2333** [Chia-Network/chia-blockchain#3934] [Q&A]
Last day the `hpool` and `uupool` (the biggest unofficial pool for now) have banned a lot of accounts as a reason of dupe-mining.
However it's more like a scam to steal coins from user as Bram worried. And it happend.

I have signed up in both pools but only actually mining in one pool and I found that I got banned, with all my coins are freezed. It may because of their operational problem but showing the horrible of centralization.

However these two pools are the only way to farm for me and many others (have a space of lower than 100TiB).
Can't wait for the official pool, and it would be nice if there is an ETA.

**Comment by oswaldzc** [comment]
I don't know if the official pool will be launched,may be a bit against decentralization.

**Comment by grandfatheryun** [comment]
> Last day the `hpool` and `uupool` (the biggest unofficial pool for now) have banned a lot of accounts as a reason of dupe-mining.
> 
> However it's more like a scam to steal coins from user as Bram worried. And it happend.
> 
> 
> 
> I have signed up in both pools but only actually mining in one pool and I found that I got banned, with all my coins are freezed. It may because of their operational problem but showing the horrible of centralization.
> 
> 
> 
> However these two pools are the only way to farm for me and many others (have a space of lower than 100TiB).
> 
> Can't wait for the official pool, and it would be nice if there is an ETA. 

opps, sorry hear that, i am try to make a really blance pool，your can fllow us ，if you have qq just jion us 658480301

**Comment by unitedchia** [comment]
The current pools are actually pretty scammy and require your private key and your mnemonic seed 24 word phrase.

The official pool protocol will be released on 05/17/2021 as stated by [Bram Cohen on Twitter](https://twitter.com/bramcohen/status/1390407484179763202).
There are already a variety of nice and fair concepts for pool farming, which do not require your private key nor your mnemonic seed 24 word phrase.

One of those pools is us. If you are interested, feel free to subscribe to our mailinglist on [chiaunited.com](https://chiaunited.com) 
The first 300 users get free pool farming for 6 months.

**Comment by donaldmcgillavry** [comment]
small farmers: waiting for pooling support
estimated time to win: 1 year... 3 years... 4 years... 5 years......
¯\\\_(ツ)\_/¯

**Reply by donaldmcgillavry** [reply]
6 years... 7 years....!

**Comment by joelrb** [comment]
There will be no ETA for pool. It will be released when it's ready.

Watch yesterday's tech briefing. This is for Pool Operators but you'll know what's the status of pools:
https://www.youtube.com/watch?v=XzSZwxowPzw
